bedrock/settings/base.py

```diff
@@ -847,3 +848,7 @@ def facebook_tab_url_lazy():
 STATSD_HOST = config('STATSD_HOST', default='127.0.0.1')
 STATSD_PORT = config('STATSD_PORT', cast=int, default=8125)
 STATSD_PREFIX = config('STATSD_PREFIX', default='bedrock')
+
+SSLIFY_DISABLE = config('DISABLE_SSL', default=True, cast=bool)
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+USE_X_FORWARDED_HOST = True
```

Comment:
Not sure if the last two lines will affect the current SCL cluster. If yes we can only enable them when `not SSLIFY_DISABLE`